### PR TITLE
typo: Double word "for"

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: ChangeLog.md [hunk 1]
```diff
@@ -641,7 +641,7 @@
 
 #### Az.Resources
 * Improve handling of providers for 'Get-AzResource' when providing '-ResourceId' or '-ResourceGroupName', '-Name' and '-ResourceType' parameters
-* Improve error handling for for 'Test-AzDeployment' and 'Test-AzResourceGroupDeployment'
+* Improve error handling for 'Test-AzDeployment' and 'Test-AzResourceGroupDeployment'
```
Comment (markcowl) [inline ChangeLog.md]: changes to earlier release notes are not accepted.

File: src/Resources/Resources/ChangeLog.md [hunk 2]
```diff
@@ -43,7 +43,7 @@
 
 ## Version 1.3.0
 * Improve handling of providers for `Get-AzResource` when providing `-ResourceId` or `-ResourceGroupName`, `-Name` and `-ResourceType` parameters
-* Improve error handling for for `Test-AzDeployment` and `Test-AzResourceGroupDeployment`
+* Improve error handling for `Test-AzDeployment` and `Test-AzResourceGroupDeployment`
```
Comment (markcowl) [inline src/Resources/Resources/ChangeLog.md]: changes to previous changelog entries are not accepted

File: src/Websites/Websites/help/Switch-AzWebAppSlot.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ The **Switch-AzWebAppSlot** switches two slots associated with an Azure Web App.
 PS C:\> Switch-AzWebAppSlot -SourceSlotName "sourceslot" -DestinationSlotName "destinationslot" -ResourceGroupName "Default-Web-WestUS" -Name "ContosoWebApp"
 ```
 
-This command will switch slot "sourceslot" slot with "destinationslot" for for Web App ContosoWebApp associated with the resource group Default-Web-WestUS
+This command will switch slot "sourceslot" slot with "destinationslot" for Web App ContosoWebApp associated with the resource group Default-Web-WestUS
```
Comment (markcowl) [inline src/Websites/Websites/help/Switch-AzWebAppSlot.md]: This needs an accompanying new entry in the associated changelog

**Pull request closed without merging**